public/docs/ts/latest/guide/setup.jade

```diff
@@ -0,0 +1,142 @@
+block includes
+  include ../_util-fns
+
+:marked
+  # Trying Angular
+
+  Live coding [Quickstart](../quickstart.html) in the browser is a great way to explore Angular.
+  Links on almost every documentation page will open completed samples in the browser where you can
+  play with the code, share your changes with friends, and download and run the code on your own machine.
+
+  As much fun as this is
+  * you can't ship your app in plunker
+  * you aren't always online when writing code
+  * transpiling TypeScript in the browser is slow
+  * the type support, refactoring, and code completion only work in your local IDE
+ 
+  # Setup a local development environment
+
+  Fortunately, setting up a new project in your local development environment is quick and easy
+  with the [**QuickStart seed**](https://github.com/angular/quickstart "Install the github QuickStart repo"),
+  a repository on github. 
+
+  Make sure you have [node and npm installed](#install-node "What if you don't have node?").
+  Then ...
+  1. [clone](#clone) or [download](#download) the **QuickStart seed repository**.
+  1. [install the `npm` packages](#install-node "What if you don't have node?")
+  1. run `npm start` to launch the sample application
+
+a#clone
+:marked
+  ### Clone

```

Comment:
Markdown sets the h3 id to "clone". By explicitly defining the anchor on line 29, this creates two document elements with the same id.